test/test_wikicorpus.jl

```diff
@@ -25,13 +25,11 @@ using InternedStrings
     #Make sure we're not catching anything we shouldn't
     @test "ENDOFARTICLE" ∉ vocab
     @test "ENDOFARTICLE." ∉ vocab
-    @test "<doc" ∉ map(x->x[1:min(end,4)], vocab)
-
-
```

Comment:
Why is this commented out?

Reply:
Might have been by mistake. I have uncommented it now.